src/util/animations.hpp

```diff
@@ -59,7 +93,7 @@ namespace big::animations
 		{
 			LOG(INFO) << "Animations file is not in directory. https://raw.githubusercontent.com/DurtyFree/gta-v-data-dumps/master/animDictsCompact.json";
```

Comment:
Make the log WARNING/ERROR to make it more obvious to users, they may not notice if the log level is just info.